.github/workflows/goreleaser.yml

```diff
@@ -30,11 +30,17 @@ jobs:
       - name: Setup Docker Buildx (docker multi-arch dependency)
         uses: docker/setup-buildx-action@v2
 
+      - name: dockerhub-login
```

Comment:
Does this deploy to docker hub every time a release tag is created? I'm not familiar with the frequency that this workflow is triggered.

Reply:
Yes, it's been triggered once so far.